application/cli-app/src/main/java/org/mobilitydata/gtfsvalidator/Main.java

```diff
@@ -81,6 +81,7 @@ public static void main(String[] args) {
 
                 // base validation + build gtfs entities
                 filenameListToProcess.forEach(filename -> {
+                    config.validateCsvNotEmptyForFile(filename).execute();
```

Comment:
I think this should only be an error for required GTFS files? And a warning for optional files? See https://gtfs.org/reference/static#dataset-files.

Reply:
It's true I interpreted an 'empty' file as a file defining zero rows of data but potentially having headers.
But the spec says
> The first line of each file must contain field names.

So I guess the only thing that shall be checked is a totally empty file (no headers), and have it being an error for any file.